src/url-fetch/UrlFetchApp.js

```diff
@@ -1,12 +1,15 @@
-const HttpResponse = require('./classes/HttpResponse')
+const UrlFetchAppStubConfiguration = require('./classes/UrlFetchAppStubConfiguration')
 
 class UrlFetchApp {
   static fetch (url, params = {}) {
-    return new HttpResponse()
+    const data = UrlFetchAppStubConfiguration.get(url)
+    if(data)
+      return data.response
+    return null
   }
 
   static fetchAll (...requests) {
-    return [...Array(requests.length)].map(() => this.fetch())
+    return requests.map(request => this.fetch(request.url, request.params))
```

Comment:
This is the breaking change. Do we definitely need not to break changes although this is something important

Reply:
@jsarafajr

Reply:
Ok, I think that's okay then, we'll just publish these changes as a new major version.